codes/data/china/collate_data.py

```diff
@@ -376,9 +376,14 @@ def log_interpolate(array):
 # source:
 # https://english.kyodonews.net/news/2020/02/6982cc1e130f-china-records-2-straight-days-of-fewer-than-1000-new-covid-19-cases.html
 # https://www.worldometers.info/coronavirus/how-to-interpret-feb-12-case-surge/
+# https://www.medrxiv.org/content/10.1101/2020.03.23.20041319v1.full.pdf
 df.loc[:, 'testing_regime'] = (
-    (df['date'] >= pd.Timestamp('2020-02-13')).astype(int) +
-    (df['date'] >= pd.Timestamp('2020-02-20')).astype(int)
+    (df['date'] > pd.Timestamp('2020-01-17')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-01-27')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-05')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-12')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-19')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-03-04')).astype(int)
```

Comment:
also, i think this change starts on 3/4? so (df['date'] >= pd.Timestamp('2020-03-04')).astype(int) + according to https://www.medrxiv.org/content/10.1101/2020.03.23.20041319v1.full.pdf. maybe i just don't know Python and this is right.

Reply:
I'm assuming that as the new guideline is published on 3/4, the new regime starts to have effects on 3/5? So it's basically discarding the log difference between 3/4 and 3/5, which is desired. lmk what you think!

Reply:
to me it isn't ex ante clear whether we should discard the log diff between 3/4 and 3/5 or between 3/3 and 3/4; seems reasonable to assume that cities take a little bit of time to respond to changes announced by the central government?

Reply:
Agree! dunno why i was being so unnecessarily picky